src/vs/workbench/parts/debug/common/debugModel.ts

```diff
@@ -624,11 +676,13 @@ export class Model extends ee.EventEmitter implements debug.IModel {
 
 	public sourceIsUnavailable(source: Source): void {
 		Object.keys(this.threads).forEach(key => {
-			this.threads[key].callStack.forEach(stackFrame => {
```

Comment:
For some reason I had missed updating this reference to `callStack`.

Just pointing out what we have changed today that wasn't in yesterday's commit and you hadn't commented on.